tlsr9/ble/vendor/controller/Kconfig

```diff
@@ -20,6 +20,19 @@ config B9X_BLE_CTRL_MAC_PUBLIC_DEBUG
 	help
 	  Enable BLE public MAC address generation
 
+config B9X_BLE_PUBLIC_MAC_ADDR
+        depends on B9X_BLE_CTRL_MAC_PUBLIC_DEBUG
+	string "Default Development Public BLE Address"
+	default "00:11:22:38:c1:a4"
+	help
+	  Development Public BLE Address
+
+config B9X_BLE_CTRL_MAC_FLASH
+	bool "MAC address flash"
+	default n
+	help
+	  One-time MAC generation and writing to FLASH
+
```

Comment:
Adding config option seems great. But I think some code is required to apply this config...
I think we discussed about the next:
`config B9X_BLE_PUBLIC_MAC_ADDR
	depends on B9X_BLE_CTRL_MAC_TYPE_PUBLIC
	string "Default Public BLE MAC Address"
	default "00:11:22:38:c1:a4"
	help
	  Vendor public BLE MAC Address

config B9X_BLE_CTRL_MAC_FLASH
	depends on B9X_BLE_CTRL_MAC_TYPE_RANDOM_STATIC
	bool "Store static BLE MAC address in flash"
	default n
	help
	  Store once generated static BLE MAC address in flash`
B9X_BLE_CTRL_MAC_PUBLIC_DEBUG will be removed...